lte/gateway/docker/docker-compose.yaml

```diff
@@ -178,6 +182,10 @@ services:
     container_name: sctpd
     ulimits:
       core: -1
+    healthcheck:
+      test: ["CMD", "bash", "-c", "[ -S /tmp/sctpd_downstream.sock ]"]
+      timeout: "5s"
```

Comment:
There is a different timeout as for everything else configured here. Why?

Reply:
It failed with the 4s timeout.

Reply:
What was the test setup that made it fail? On my machine it worked with a timeout of one second. To be fair I only started sctpd and not the other containers, but then again a simple bash test command should usually take way less than a second I think.

Reply:
nitpick: We could also write `test: "test -S /tmp/sctpd_downstream.sock"`, personally I find that more readable, especially the combination of square brackets for Yaml lists and for the bash `test` command are a bit hard to parse.

Reply:
It worked for me as well with 4s (I tested starting all containers).